main/FatesHydraulicsMemMod.F90

```diff
@@ -14,8 +14,8 @@ module FatesHydraulicsMemMod
    ! Define the various different solver options for hydraulics
    
    integer, parameter, public :: hydr_solver_1DTaylor = 1
-   integer, parameter, public :: hydr_solver_2DNewton = 2
-   integer, parameter, public :: hydr_solver_2DPicard = 3
+   integer, parameter, public :: hydr_solver_2DNewton = 3
```

Comment:
Why did this change?  These are flipped from the comment description in EDParamsMod.

Reply:
Yeah, good point.  The Picard solver is now the preferred method of performing a 2D matrix type solve, and the Newton is somewhat deprecated.  So I changed the order to reflect that, the order is consistent with the metadata in the parameter file

Reply:
oh, ok, didn't realize that it was not consistent with EDParamsMod, will update that text